libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -973,3 +973,9 @@ singleton AP_VisualOdom depends HAL_VISUALODOM_ENABLED
 singleton AP_VisualOdom rename visual_odom
 singleton AP_VisualOdom method healthy boolean
 singleton AP_VisualOdom method quality int8_t
+
+-- ----AP_TemperatureSensor Library----
+include AP_TemperatureSensor/AP_TemperatureSensor.h
+singleton AP_TemperatureSensor depends AP_TEMPERATURE_SENSOR_ENABLED
+singleton AP_TemperatureSensor rename temperature_sensor
+singleton AP_TemperatureSensor method get_temperature boolean float'Null uint8_t 0 ud->num_instances()
```

Comment:
We might need to add some checks in `AP_TemperatureSensor` but we have been trying to not limit instances by `ud->num_instances()` like this. The problem is that a script has no way to know `ud->num_instances()` at runtime. A too large value will cause the script to error out (unless you pcall).

Reply:
Looks like its safe to call with any value. 
```suggestion
singleton AP_TemperatureSensor method get_temperature boolean float'Null uint8_t'skip_check
```

Reply:
Done and re-tested, thanks!